### ISSUE TITLE
 SyntaxError: Unexpected token import

### ISSUE THREAD

**Issue by JeffWScott** [RobinGulbrandsen/react-blog#9]
I came here from a stack overflow thread.  This page was used as an example to properly setup routes.  I liked the way this was done so I set about converting my routing to this Controller stuff.

Unfortunately I hit a snag right away with importing my controller file.  Any idea why it doesn't like the word "import"?! 

I'm using node v6.11.1
'./controllers/Programs' does certainly exists.

> (function (exports, require, module, __filename, __dirname) { import ProgramsController from './controllers/Programs';
>                                                               ^^^^^^
> SyntaxError: Unexpected token import

here is the entirety of my routes file:

> import ProgramsController from './controllers/Programs';
> 
> module.exports = (app) => {
> 
>   //////// ARTICLES ////////////
>   app.get('/programs/all',        ProgramsController.getAll);
> 
> };

**Comment by RobinBorresen** [comment]
tl;dr; 'import' is ES6 syntax equivalent to 'require'.

```
import Controller from './Controller.js'
```
vs.
```
const Controller = require('./Controller');
```

It's some time since I worked on this code now, but I seem to remember that Node 9 has support for this. The code in question is currently running on http://rgulbrandsen.com/#/ . This comment is mostly to let you know the reason for the issue and to let you know that I'll change the syntax to the Node 6 friendly way with 'require' within a few days.

Thank you for your comment!

**Comment by JeffWScott** [comment]
Makes sense.  Thanks Robin!
